docs/source/en/basics/router.md

```diff
@@ -0,0 +1,328 @@
+app.get('/user/:id', 'user.page');
+app.post('/admin', isAdmin, 'admin');
+app.post('user', '/user', isLoginUser, hasAdminPermission, 'user.create');
+```
+
+### RESTful style URL definition
+
+We provide `app.resources('router-name', 'path-match', 'controller-name')` to produce [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) structures on a path for convenient if you prefer the RESTful style URL definition.
+
+```js
+// app/router.js
+module.exports = app => {
+  app.resources('posts', '/posts', 'posts');
+};
+```
+
+The codes above produces a bunch of CRUD path structures for Controller `app/controller/posts.js`, and the only thing you should do next is to implement related functions in `posts.js`.
+
+Method | Path            | Route Name     | Controller.Action
+-------|-----------------|----------------|-----------------------------
+GET    | /posts          | posts          | app.controllers.posts.index
+GET    | /posts/new      | new_post       | app.controllers.posts.new
+GET    | /posts/:id      | post           | app.controllers.posts.show
+GET    | /posts/:id/edit | edit_post      | app.controllers.posts.edit
+POST   | /posts          | posts          | app.controllers.posts.create
+PUT    | /posts/:id      | post           | app.controllers.posts.update
+DELETE | /posts/:id      | post           | app.controllers.posts.destroy
+
+```js
+// app/controller/posts.js
+exports.index = function* () {};
+
+exports.new = function* () {};
+
+exports.create = function* () {};
+
+exports.show = function* () {};
+
+exports.edit = function* () {};
+
+exports.update = function* () {};
+
+exports.destroy = function* () {};
+```
+
+Methods that are not needed may not be implemented in `posts.js` and the related URL paths will not be registered to Router neither.
+
+## destroy in action
+
+The destroy syntax will demonstrated by more examples.
+
+#### Acquiring Parameters
+
+#### via Query String
+
+```js
+// app/router.js
+module.exports = app => {
+  app.get('/search', 'search');
+};
+
+// app/controller/search.js
+module.exports = function* (ctx) {
+  ctx.body = `search: ${this.query.name}`;
+};
+
+// curl http://127.0.0.1:7001/search?name=egg
+```
+
+#### via Named Parameters
+
+```js
+// app/router.js
+module.exports = app => {
+  app.get('/user/:id/:name', 'user.info');
+};
+
+// app/controller/user.js
+exports.info = function* (ctx) {
+  ctx.body = `user: ${ctx.params.id}, ${ctx.params.name}`;
+};
+
+// curl http://127.0.0.1:7001/user/123/xiaoming
+```
+
+#### acquiring complex parameters
+
+Regular expressions, as well, can be used in routing rules to aquire parameters more flexible:
+
+```js
+// app/router.js
+module.exports = app => {
+  app.get(/^\/package\/([\w-.]+\/[\w-.]+)$/, 'package.detail');
+};
+
+// app/controller/package.js
+exports.detail = function* (ctx) {
+  // If the request URL is matched by the regular expression, parameters can be get from ctx.params according to the capture group orders.
```

Comment:
accept.

I think both 'get' and 'acquired' are okay: 'get' is brief and oral while 'acquired' is technical.